src/CreaTeBME/SensorEmulator.py

```diff
@@ -26,13 +27,18 @@ def __init__(self, filename: str):
         self._callback = None
         self._is_running = False
 
+        self._start_time = None
+        self._iter = 0
```

Comment:
Could you change iter (probably iterator) to be `counter`, `frame_counter` or even `iterator` spelled out. Or add a comment cause `iter` is a bit vague what it really does.